### PR TITLE
Bianca

### PR DESCRIPTION
**Pull request by siasktv**
Landing Page & Navbar

### CODE REVIEW

File: client/dummy-crypto-data/marketTrendsCrypto.js
Comment (keldim) [inline client/dummy-crypto-data/marketTrendsCrypto.js:1]: hey bianca, where is this file being used?
would it be okay to delete it if it is not being used?

File: client/assets/Logo.png
Comment (keldim) [inline client/assets/Logo.png:1]: hey bianca, I think it is better to put the 'assets', 'data', 'dummy-crypto-data' folders inside the 'src' folder
just to be safe
https://stackoverflow.com/questions/44643041/do-i-store-image-assets-in-public-or-src-in-reactjs

**Review by yi-lin-1234 — APPROVED** [review]
worked as expected

**Review by keldim — CHANGES_REQUESTED** [review]
left comments in the code

File: client/dummy-crypto-data/marketTrendsCrypto.js
Comment (siasktv) [reply client/dummy-crypto-data/marketTrendsCrypto.js:1]: yes, it can be delete it

File: client/assets/Logo.png
Comment (siasktv) [reply client/assets/Logo.png:1]: sure!

**Review by keldim — APPROVED** [review]

**Pull request merged**